src/dart_dll.cpp

```diff
@@ -146,17 +146,22 @@ bool DartDll_Initialize(const DartDllConfig& config) {
 }
 
 Dart_Isolate DartDll_LoadScript(const char* script_uri,
-                                const char* package_config) {
+                                const char* package_config,
+								void* callbackPtr) {
```

Comment:
Can we change the name of this to "isolate_data" or similar? Currently the name makes it sound like it should be a function.

Reply:
Can do, just reused the name for the existing parameter.